woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -358,6 +358,19 @@ def test_accessor_equality(sample_df):
         assert schema_df.ww == iloc_df
 
 
+def test_accessor_equality_make_index(sample_df_pandas):
+    made_index_df = sample_df_pandas.copy()
+    made_index_df.insert(0, 'new_index', range(len(made_index_df)))
+    made_index_df.ww.init(index='new_index', make_index=False)
+
+    schema_df = sample_df_pandas.copy()
+    schema_df.ww.init(index='new_index', make_index=True)
+
+    pd.testing.assert_frame_equal(schema_df, made_index_df)
+
+    assert made_index_df.ww != schema_df
```

Comment:
I think there is a missing `ww` on `schema_df`:  `assert made_index_df.ww != schema_df.ww`

Reply:
yep!